### DISCUSSION TITLE
Random crash and can not scroll with trackpad

### DISCUSSION THREAD

**Discussion by xxleyi** [johnlindquist/kit#768] [General]
After upgraded to 1.17.2
- kit occasionally quited in background 
- can not scroll in kit prompt with trackpad

**Comment by johnlindquist** [comment]
I need a lot more info to help:

Which version of osx are you on?
How often does the random crash happen?
Are you running any // Background or // Schedule scripts?
Did you grant Kit.app accessibility permissions when it prompted?



Which UI element won't scroll?

**Reply by xxleyi** [reply]
> I need a lot more info to help:
> 
> Which version of osx are you on? How often does the random crash happen? Are you running any // Background or // Schedule scripts? Did you grant Kit.app accessibility permissions when it prompted?
> 
> Which UI element won't scroll?

- MacOS version 10.15.7
- four or five times one day
- no background or schedule script
- yes, granted in old version
![image](https://user-images.githubusercontent.com/12626454/167626855-151e6f07-3384-4514-9a01-bdc2a2f16a7d.png)

About scroll issue, I just tested again, seemed work well, however still have an issue: when kit prompts,  `hs.application.watcher`(hammerspoon) no longer detect its activated event, which worked in 1.11.1 and older version,  I used this to auto switch input method to English.

**Reply by johnlindquist** [reply]
Hmm, could you join our Discord server? This would be easier to chat about and I could send you a private beta that might help fix:

https://discord.gg/hvM8Vaex

**Reply by xxleyi** [reply]
> Hmm, could you join our Discord server? This would be easier to chat about and I could send you a private beta that might help fix:
> 
> https://discord.gg/hvM8Vaex

okay
